README.md

```diff
@@ -1,29 +1,42 @@
 # MyProject
 ## 📖 About the Project
-This project is about Application Deployment, Here is the link: https://hirwacardinalloic.github.io/MyProject/
+This project we built  POPUP Website
 
 ## 🛠 Built With
 -HTML 
 -CSS
+-JavaScript
 
 ## 💻 Getting Started
 To get a local copy up and running, follow these steps.
 1. Open Terminal
 2. Clone the Project 
 3. Open browser using `live server`
 
+## live Demo
+ - Deployment : https://hirwacardinalloic.github.io/MyProject/
+
+
 ## Prerequisites
 - IDE to edit and run the code (We use Visual Studio Code).
 - Git to versionning your work.
 
 ## 👥 Authors
  - GitHub: [@Hirwacardinalloic]
  - Twitter: [@loicard]
+
+## Key Feature
+ - To guide those in need
+
 ## 🔭 Future Features
+ - am going to add contact validation
+
 ## 🤝 Contributing
 Contributions, issues, and feature requests are welcome!
+
 ## ⭐️ Show your support
 Give a ⭐️ if you like this project
+
 ## 🙏 Acknowledgements
 - Microverse program 
 - My standup team 
```

Comment:
Hi @Hirwacardinalloic 
You have done a great job so far but unfortunately, you are still using the old README file 
Please kindly follow the new Microverse [README template](https://github.com/microverseinc/readme-template) file and update your README file.